### ISSUE TITLE
FightPandemic - Frontend Engineer - Remote Worldwide

### ISSUE THREAD

**Issue by letizia123** [TechnologyMasters/jobs#324]
### Salary Expectation

- Volunteer

### Benefits

Why Join Us?

- Expand your network. You get to work as a part of a innovative, global and diverse team.
- Be part of a movement. We strive for bold changes and envision to make a big impact. Bring your passion, energy and creativity to participate.
- Learn and grow. You get to continuously learn and develop your skills. You might join us because of your skills in business development, but are able to contribute to any team's work if you'd like.
- Be free. You have the freedom to set the hours you are able to work every week.
- Do good. It is a scientific fact that volunteering for a good cause improves your well being.
 
### Location

- Remote 

## What You'll Do _(Job Description)_

- Develop features and improvements to the Fight Pandemics product in a secure, well-tested, and performant way.
- Craft code that meets our internal standards for style, maintainability, and best practices for mobile responsiveness, cross-browser compatibility, and strong accessibility.
- Collaborate with other engineers, product managers and other stakeholders (e.g. UX) to maintain a high bar for quality in a fast-paced, iterative environment.
- Take ownership of the complete lifecycle of a product features from planning to implementation to testing to production.

### Must Have

- You are passionate about what we are building and you want to contribute and not wait on the sideways.
- You are positive and have a solution-oriented mindset.
- You are a problem solver and enjoy writing robust code
- You have attention to detail with strong communication skills.
- Knowledgable about building full-stack web applications using HTML5, CSS3, and JavaScript related technologies such as React and Node.js.
- Knowledgable about building APIs that are RESTful.
- Knowledgeable about SQL and NoSQL databases and building efficient data queries and pipelines.
- Knowledgeable about building secure APIs that are not vulnerable to security attacks.
You know how to use basic Git commands.
- Most importantly, you want to be part of something great that positively impacts society especially in times of crisis.

### Nice to Have

- Experience working with our stack
- Frontend: React Hooks, Ant Design, Ant Design Mobile, styled-components.
- Experience building UI that is mobile responsive, cross-browser compatible, and has strong accessibility.
- Experience with mobile development.

### Relevant Experiences

- Openness to learn new technologies.
- Experience working with Git and Github in teams.
- Experience working with a remote team.
- Open source experience.
- Developer platform/tool industry experience

## About Our company

We are creating an altruistic platform that facilitates the connection and exchange of information between individuals and/or organizations that need help with those that are providing help and vice versa. The way that we are going to bootstrap our social media, is by listing thousands of projects and initiatives that are tackling COVID-19 pandemic.
We are looking for volunteer who want to join us in our fight against pandemics like COVID-19 and other crisis alike.

## How to apply

Please apply on our website -> https://apply.workable.com/fightpandemics/j/A5C2F28F3E/
---

## Meta

- [ ] Full Time
- [ ] Part Time
- [ ] Contract
- [ ] Internship
- [x] Remote Worldwide
- [ ] Remote Regional
- [x] Remote OK

**Comment by theetrain** [comment]
Closing due to missing information.
